src/02_spec/07_modules/dem_uart/systemif.rst

```diff
@@ -0,0 +1,53 @@
+System Interface
+----------------
+
+The following signals are required from a bus in order to connect to the DEM.
+
++-----------------+----------+---------------+-------------------------------------------------+
+| Signal          | Width    | Direction     | Description                                     |
++=================+==========+===============+=================================================+
+| ``bus_req``     |    1     | CPU->DEM      | ``1`` indicates that slave is currently selected|
++-----------------+----------+---------------+-------------------------------------------------+
+| ``bus_addr``    |    3     | CPU->DEM      | Address to be used with write/read operation    |
++-----------------+----------+---------------+-------------------------------------------------+
+| ``bus_write``   |    1     | CPU->DEM      | ``1`` = read, ``0`` = write                     |
```

Comment:
Really? The signal is called bus_write but a '1' indicates a read? If that is the case, please rename the signal.